app/MMB/src/components/Alert.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { Text, StyleSheet, Icon } from '@kiwicom/mobile-shared';
+import { Translation } from '@kiwicom/mobile-localization';
+
+type CommonProps = {|
+  title: React.Element<typeof Translation>,
+  children: React.Node,
+|};
+
+type Props = {|
+  type: 'danger' | 'warning' | 'success',
+  ...CommonProps,
+|};
+
+const AlertDanger = (props: CommonProps) => (
+  <View style={[styles.container, styles.containerDanger]}>
+    <View style={styles.iconContainer}>
+      <Icon name="warning" size={18} color={RED} />
+    </View>
+    <View>
+      <Text style={[styles.titleText, styles.titleDanger]}>{props.title}</Text>
+      <View style={styles.body}>{props.children}</View>
+    </View>
+  </View>
+);
+
+export default function Alert(props: Props) {
+  switch (props.type) {
+    case 'danger':
+      return <AlertDanger title={props.title}>{props.children}</AlertDanger>;
+    case 'warning':
+      // TODO
+      return null;
+    case 'success':
+      // TODO
+      return null;
+    default:
+      return null;
+  }
+}
+
+const RED = '#d0021b';
```

Comment:
Should we use new brand colors here or not? https://zpl.io/2GmG6mW

![screen shot 2018-05-23 at 09 52 43](https://user-images.githubusercontent.com/978368/40410900-2e0175f8-5e6f-11e8-9d05-49530d9f1096.png)

Reply:
🤷‍♂️ are there any designs for these `alerts` in the new design?

Reply:
Only this: https://zpl.io/bzYGpjz

The colors are different though...

Reply:
The color codes in the new `alerts` are the same as before